src/Components/HomePage/Homepage.test.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { render, screen, waitFor } from "@testing-library/react";
+import { Homepage } from "./Homepage";
+import { MemoryRouter } from 'react-router-dom'
+import "@testing-library/jest-dom";
+
+describe("BrowsePage", () => {
+    it("User should see links to view different activity categories", () => {
+      render(<MemoryRouter><Homepage /></MemoryRouter>);
```

Comment:
I am no sure it is this view or the linter's doing, but I think it would be better to indent line 9, 18, 23 and 32.

Reply:
Yeah I think it'd be a little more readable that way, but I know it doesn't affect the way the test functions.